llvm/test/tools/llvm-symbolizer/approximate-line-info.s

```diff
@@ -0,0 +1,153 @@
+# REQUIRES: x86-registered-target
```

Comment:
Probably needs more test coverage, it doesn't look like this distinctly tests the "stop at prologue end even if we didn't find a non-zero line" (that'd need a prologue end with a zero line) - if that feature's going to stay in this patch, that is.

Reply:
No longer required since approximation upto BasicBlock and Function Boundaries is omitted from current revision.